_stbt/ocr.py

```diff
@@ -631,12 +631,7 @@ def _log_ocr_image_debug(imglog, text):
 
         <div class="annotated_image">
           <img src="source.png" />
```

Comment:
We should probably set `width="{{image.width}}"` and `height="{{image.height}}"` here so the browser can do layout before loading the image.

Reply:
I see, maybe we can't do that or it will break the `max-width` bit.